Plugins/Trivia/TriviaPlugin.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Trivia
 {
-    record PersistentDataStructure(double score, double numQest, int trueResult);
```

Comment:
Fixing the bug is not related to this PR
You know how to work with branches?

Reply:
no

Reply:
I recommend learning this...
search "git branches"...

Reply:
OK. What about the current one's approval?

Reply:
For now, I will approve